common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -316,9 +316,10 @@ sNodeBssSteerTarget:
 sNodeBssSteerRequest:
   _type: struct
   mac: sMacAddr
-  disassoc_timer: uint16_t
+  bssid: sMacAddr
```

Comment:
To avoid confusion about this, I would rename this to `source_bssid` or `cur_bssid`.